### ISSUE TITLE
Place of clamav-unofficial-sigs.log

### ISSUE THREAD

**Issue by takenek** [extremeshok/clamav-unofficial-sigs#46]
hey
I just install new 5.0 version of clamav-unofficial-sigs on Debian Jessie and get error when i run from hand debug version:
/usr/local/bin/clamav-unofficial-sigs.sh: linia 79: /var/log/clamav-unofficial-sigs/clamav-unofficial-sigs.log: Nie ma takiego pliku ani katalogu

mayby better change place of log file to /var/log/ or do check in clamav-unofficial-sigs.sh if directory /var/log/clamav-unofficial-sigs/ not exist to create new one?

Thanks.

**Comment by extremeshok** [comment]
Will do.

Please let me know if u have any other issues.

**Comment by sts** [comment]
Ran into the same issue. I configured the logfile to ```/var/log/clamav-unofficial-sigs.log``` and its still trying to log to ```/var/log/clamav-unofficial-sigs/clamav-unofficial-sigs.log```. I think the logfile path is evaluated after its trying to log the first time. @extremeshok

**Comment by extremeshok** [comment]
Fixed in 5.0.1 by commits 381ae55559d738a0d22d5cb1717bf3d7a9b97df1 and e953372d91bc325f1a2a3d7b17759af703159a17

**Comment by sts** [comment]
Not completely. You allow the user to configure a logfile path, but you are logging before you evaluate the configuration, and thus you always try to log to a default file location, whenever the script starts.

```
################################################################################
 eXtremeSHOK.com ClamAV Unofficial Signature Updater
 Version: v5.0.0 (23 March 2016)
 Required Configuration Version: v55
 Copyright (c) Adrian Jon Kriel :: admin@extremeshok.com
################################################################################
=======================================================
Loading config: /etc/clamav-unofficial-sigs/master.conf
=======================================================
===================================================
Loading config: /etc/clamav-unofficial-sigs/os.conf
===================================================
/usr/sbin/clamav-unofficial-sigs: line 79: /var/log/clamav-unofficial-sigs/clamav-unofficial-sigs.log: No such file or directory
=====================================================
Loading config: /etc/clamav-unofficial-sigs/user.conf
=====================================================
==================================================
Sanesecurity Database & GPG Signature File Updates
==================================================
Sanesecurity mirror site used: rsync-mirror.rollernet.us 208.79.241.67

Number of files: 50 (reg: 50)
Number of created files: 0
Number of regular files transferred: 0
Total file size: 21,364,072 bytes
Total transferred file size: 0 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 1,679
File list generation time: 0.354 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 34
Total bytes received: 2,022

sent 34 bytes  received 2,022 bytes  587.43 bytes/sec
total size is 21,364,072  speedup is 10,391.09
-------------------------------------------
No Sanesecurity database file updates found
-------------------------------------------
```

After the first error, logging works because you evaluated the user.conf.

**Comment by extremeshok** [comment]
Fixed in 5.0.1

**Comment by extremeshok** [comment]
and 226e4a7e4c88493a557e3185e4dcdce2d7490ed3
